src/main/java/com/descope/model/magiclink/request/UpdateEmailRequest.java

```diff
@@ -11,12 +11,14 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class UpdateEmailRequest {
-
   private String email;
   private String loginId;
-
   @JsonProperty("URI")
   private String uri;
-
   private boolean crossDevice;
+  @JsonProperty("addToLoginIDs")
+  private boolean addToLoginIds;
+  private boolean onMergeUseExisting;
+  private String providerId;
```

Comment:
noting that only this sdk will now support providerId and templateId inputs

@Bars92 fyi

Reply:
I removed it as per Aviad's comment